packages/js/ai/src/hooks/useCompletion.ts

```diff
@@ -9,6 +9,7 @@ import { useRef, useState } from '@wordpress/element';
 import { getCompletion, createExtendedError } from '../utils/';
 
 type StopReason = 'abort' | 'finished' | 'error' | 'interrupted';
+type ResponseFormat = 'json_object' | 'text';
```

Comment:
```suggestion
export type ResponseFormat = 'json_object' | 'text';
```

What do you think about exporting this type so that it can be imported by text-completion.ts?

Reply:
Yeah, that makes sense. I added a shared `types.ts` file to store these types in for reuse.